internal/notification/handlers/user_notifier.go

```diff
@@ -259,66 +299,70 @@ func (u *userNotifier) reducePasswordCodeAdded(event eventstore.Event) (*handler
 		if alreadyHandled {
 			return nil
 		}
-		var code string
-		if e.Code != nil {
-			code, err = crypto.DecryptString(e.Code, u.queries.UserDataCrypto)
-			if err != nil {
-				return err
-			}
-		}
-		colors, err := u.queries.ActiveLabelPolicyByOrg(ctx, e.Aggregate().ResourceOwner, false)
+		ctx, err = u.queries.Origin(ctx, e)
 		if err != nil {
 			return err
 		}
+		origin := http_util.DomainContext(ctx).Origin()
+		urlTmpl := e.URLTemplate
+		if urlTmpl == "" {
```

Comment:
This condition could be removed if you have a general if empty in the function right?